dashbuilder-client/dashbuilder-navigation-client/src/main/resources/org/dashbuilder/client/navigation/resources/i18n/NavigationConstants.properties

```diff
@@ -31,34 +31,45 @@ navItemNotFound="Not found: {0}
+navWidgetDeadlockError=The content is not displayed to avoid an infinite loop. This is likely caused by a circular reference in the navigation configuration.
+navWidgetTargetDivMissing=The target div is either not found or not defined. Please, review the component''s configuration.
 openNavItem=Open {0}
 gotoNavItem=Go to {0}
 showNavItem=Show {0}
 navConfigHeader=Navigation component configuration
 navGroupLabel=Navigation group
-navGroupHelp=The group that will be used as the root element of the navigation component.
+navGroupHelp=The group used as the root element of the navigation component.
 navGroupSelectorHint=-Select group -
 defaultItemLabel=Default item
-defaultItemHelp=The item (e.g. a perspective) the navigation component displays by default. If nothing selected then the first item found is taken.
+defaultItemHelp=The item (e.g. a perspective) the navigation component displays by default. If nothing selected then the first item is taken.
 defaultItemSelectorHint=-Select default item -
 defaultItemsNotFound=The selected group has no items
 targetDivLabel=Target div
 targetDivHelp=The div component beloging to the current layout where a runtime perspective item is shown once clicked.
 targetDivSelectorHint=-Select target div -
 targetDivsNotFound=No target divs defined
-navItemEditorPerspectiveHelp=The perspective to display when this navigation item is clicked from any navigation component: menu bar, tile navigator, tab list, ...
-navItemEditorGroupHelp=The navigation group used by the perspective''s navigation components. That means the navigation group configured in any navigation component belonging to the perspective selected above will be overwritten by this one. Notice, this setting is only enabled for user created perspectives.
-
+navItemEditorPerspectiveHelp=The perspective to display when this navigation item is clicked from any navigation component: menu bar, tile navigator, tab list, tree, ...
+navItemEditorGroupHelp=The navigation group to be used by the target perspective''s navigation components. This setting only makes sense when perspective selected above contains navigation components.
+navRefPerspective=The perspective {0}
+navRefPerspectiveDefault=The item points to the perspective {0}.
+navRefPerspectiveFound=The item points to the perspective {0}.
+navRefPerspectiveInGroup=The group contains at least one item linked to the perspective {0}.
+navRefComponent=contains a {0}
+navRefGroupDefined=configured with the {0} navigation group.
+navRefGroupContext=receiving the {0} navigation group.
+navRefDefaultItemDefined=The default item defined is {0}.
+navRefDefaultItemFound=The first available item found is {0}.
+navRefPerspectiveDeadlockEnd=which causes a deadlock
```

Comment:
I'd avoid deadlock in UI. Maybe "which would cause infinite loop"?

Reply:
Done